### PR TITLE
Add a MicroPython course

### PR DESCRIPTION
**Pull request by encukou**
This:
* Starts by rebasing #401 by @frenzymadness 
* Splits the content to lessons rather than subpages (subpages are more for parallel material, e.g. Linux/Windows/Mac installations)
* Adds a few more lessons

The resulting course is available at `/course/micropython/`, but not yet linked anywhere. The content is usable in instructor-led courses, but shouldn't yet be shown to self-learners.

On this PR I'm requesting review for the content: is this an improvement? Will it be usable in instructor-led courses?
Please save discussions on where to put it within naucse for elsewhere.

Fixes: https://github.com/pyvec/naucse.python.cz/issues/418

### CODE REVIEW

File: lessons/micropython/motors/index.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+Motory potřebují, na rozdíl od počítače a LED světýlek, celkem hodně elektrické
+energie, a navíc můžou dokonce energii vyrábět (fungují jako dynamo).
+Kdybys je připojil{{a}} přímo k destičce, která na tolik proudu není
+připravená, mohla by se destička zničit.
+
+Představ si náramkové hodninky a traktor: obě zařízení něčím točí (ručičkami
+nebo koly), ale kdybys připojil{{a}} motor z traktoru na mechanismus hodinek,
+moc dlouho by správný čas neukazovaly.
+A motůrek z hodinek by zase nepohohl při orání pole.
```
Comment (frenzymadness) [inline lessons/micropython/motors/index.md]: „nepohohl“ → „nepomohl“

File: lessons/micropython/motors/index.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+Proto použijeme čip s názvem L293D, který elektřinu potřebnou pro “hrubou sílu”
+motorku odstínit od logických signálů z destičky.
```
Comment (frenzymadness) [inline lessons/micropython/motors/index.md]: „odstinit“ → „odstíní“

File: lessons/micropython/motors/index.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+Potřebnou energii dodáme z baterií.
+
+Čip je černá krabička, která na sobě má trochu textu, ale ne dost na to,
+abys poznal{{a}} co dělá.
+To je deteilně popsáno v takzvaném *datasheetu* – PDF, které vypadne když
```
Comment (frenzymadness) [inline lessons/micropython/motors/index.md]: „deteilně“ → „detailně“

Comment (frenzymadness) [reply lessons/micropython/motors/index.md]: + možná čárka za „které vypadne“

File: lessons/micropython/motors/index.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+zadáš „L293D“ do vyhledávače.
+Tam lze najít kompletní popis této součástky včetně diagramy, který ukazuje
```
Comment (frenzymadness) [inline lessons/micropython/motors/index.md]: „diagramy“ → „diagramu“

Comment (frenzymadness) [reply lessons/micropython/motors/index.md]: možná čárka za „který ukazuje“

File: lessons/micropython/thermometer/index.md [hunk 5]
```diff
@@ -1,27 +1,33 @@
-## Teploměr
+# Teploměr
 
 Poslední součástkou, kterou si dnes ukážeme, bude jednoduchý teploměr DS18B20.
```
Comment (frenzymadness) [inline lessons/micropython/thermometer/index.md]: Možná upravit odstavec, aby jeho obsah nezávisel na pořadí použití.

File: lessons/micropython/motors/index.md (hunk 3 above)
Comment (encukou) [reply lessons/micropython/motors/index.md]:
```suggestion
To je detailně popsáno v takzvaném *datasheetu* – PDF, které vypadne, když
```

File: lessons/micropython/motors/index.md (hunk 2 above)
Comment (encukou) [reply lessons/micropython/motors/index.md]:
```suggestion
motorku odstíní od logických signálů z destičky.
```

File: lessons/micropython/motors/index.md (hunk 1 above)
Comment (encukou) [reply lessons/micropython/motors/index.md]:
```suggestion
A motůrek z hodinek by zase nepomohl při orání pole.
```

File: lessons/micropython/motors/index.md (hunk 4 above)
Comment (encukou) [reply lessons/micropython/motors/index.md]:
```suggestion
Tam lze najít kompletní popis této součástky včetně diagramu, který ukazuje
```

File: lessons/micropython/thermometer/index.md (hunk 5 above)
Comment (encukou) [reply lessons/micropython/thermometer/index.md]:
```suggestion
Ukážeme si jednoduchý teploměr DS18B20.
```

**Comment by encukou** [comment]
This is now in https://github.com/encukou/naucse-micropython

**Pull request closed without merging**